site/content/internal/release-process.md

```diff
@@ -16,11 +16,13 @@ On code complete day, work with the PM on rotation to get all the pull requests
 1. Give yourself access to matterbuild by adding yourself in the platform-private repo. See an [example here](https://github.com/mattermost/platform-private/commit/89f91d81bd4602f4708270c0ca7626da8fc45291). You will need to know your user Id in the community server.
 2. Trigger the matterbuild Jenkins job to update it https://build.mattermost.com/job/matterbuild.
 3. Submit a PR to uncomment the upgrade code for the release version and add it to the version array. Use these PRs as examples, [https://github.com/mattermost/mattermost-server/pull/6336/files](https://github.com/mattermost/mattermost-server/pull/6336/files) and [https://github.com/mattermost/mattermost-server/pull/6600/files](https://github.com/mattermost/mattermost-server/pull/6600/files).
-4. Once the above PR is merged, post `/mb cut X.X.X-rc1` into a private channel. Replace `X.X.X` with the release version, ex: `3.10.0`. This will begin cutting the build and make an automatic post to the Release Discussion channel.
-5. Wait approximately 25 minutes for the build to finish. If the build fails, you will need to delete the version tags from the mattermost-server, mattermost-webapp and enterprise repositories by running `git push origin :vX.X.X-rc1` in all of them. Then simply repeat step 4. You can monitor build status from https://build.mattermost.com.
+4. Once the above PR is merged, post `/mb cut X.X.X-rc1` into a private channel. Replace `X.X.X` with the release version, ex: `5.8.0`. This will begin cutting the build and make an automatic post to the Release Discussion channel.
+  - If you need to build an older release (older than 5.7.X), you will need to set the `legacy` and the `backport` flags in order to trigger the old job, ie. `/mb cut 5.7.1-rc1 --backport --legacy`.
+  - Also, for the current job, you can define the Docker images using flags to build/test server and webapp. ie `/mb cut 5.8.0-rc1 --server mattermost/mattermost-build-server:dec-7-2018 --webapp mattermost/mattermost-build-webapp:oct-2-2018`.
+5. Wait approximately 25 minutes for the build to finish. If the build fails, please check the jenkins job to see what happened. Then simply repeat step 4. You can monitor build status from https://build.mattermost.com.
```

Comment:
@cpanato maybe add a note mentioning to contact you if they are not sure why the build failed

Reply:
done @jwilander